docs/tutorials/elasticdl_local.md

```diff
@@ -6,11 +6,13 @@ the working process of ElasticDL.
 
 ## Environment preparation
 
-Here we should install Minikube first. Please refer to the official
+1. Install Minikube first. Please refer to the official
 [installation guide](https://kubernetes.io/docs/tasks/tools/install-minikube/).
```

Comment:
In this tutorial, minikube version is `v1.11.0`

Reply:
Good catch, updated.